app/controllers/albums.js

```diff
@@ -0,0 +1,16 @@
+const { getAlbums, getPhotos } = require('../services/albums');
+
+exports.listAlbums = (req, res, next) => {
+  getAlbums()
+    .then(response => res.status(200).send(response.data))
```

Comment:
when you do res.send(something) you dont need to put the 200, is set by default in Express